L4D2/Change Log.txt

```diff
@@ -1,3 +1,25 @@
+/===============================================/
+/=====================v1.7.1====================/
+/===============================================/
+18/12/2020 (ImAciidz)
+-Added support for version 2.2.1.1, removed support for 2.2.0.9
+
+/===============================================/
+/=====================v1.7======================/
+/===============================================/
+11/12/2020 (ImAciidz)
+-Added double splitting prevention for finales
+-Readded support for version 2.1.4.7
+-Added support for version 2.0.6.3
+-Added support for The Last Stand versions 2.2.0.3, 2.2.0.7, and 2.2.0.9
+-Changed "whatsLoading" variable offset for 2.0.7.5 & 2.0.9.1 as I discovered it changed dynamically during the level load as various level objects were loaded in. Likely will not affect anything, my main motivation for changing this was just because the variables selected for other versions stayed static during level loading
```

Comment:
The reason the variable is called "whatsLoading" is because it shows several objects that were loaded in. Changing the offset to prevent makes the variable name choice non-sensical. I don't think there's much use for knowing which object is loading (I just thought it was neat really, so I kept it in), so it's probably a better idea to rename the variable to something more accurate. Idk if these offsets show more than the map name at any given point, but a new name should probably refer to something like that

Reply:
I reverted the change that I made since I guess it hurts nothing to leave it the way it was.